string TYPE=loc

uint32 value
